src/main/java/org/kohsuke/github/GHMemberChanges.java

```diff
@@ -0,0 +1,52 @@
+package org.kohsuke.github;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import org.kohsuke.github.internal.EnumUtils;
+
+/**
+ * Changes made to a team.
+ */
+@SuppressFBWarnings(value = { "UWF_UNWRITTEN_FIELD" }, justification = "JSON API")
+public class GHMemberChanges {
+
+    private FromToPermission permission;
+
+    /**
+     * Get changes to permission.
+     *
+     * @return changes to permission
+     */
+    public FromToPermission getPermission() {
+        return permission;
```

Comment:
@gsmet 
Ugh.  The events api is all over the place.  The `added` action has a `role_name` and states that `permission` is obsolete.  But the `edited` action has `permission` and `old_permission`, with no note about either of those being obsolete.

https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=added#member
https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=edited#member

Reply:
I did some tests for permission and old_permission wasn’t there. 

I need to check for role_name though. I will have a look.